pylabnet/gui/igui/iplot.py

```diff
@@ -241,6 +241,7 @@ def _allocate_arrays(self, num_arrays=None):
                     )
                 self._num_ch += 1
 
+
```

Comment:
@mkbhaskar It is probably a typo.

Reply:
@mkbhaskar Sorry - probably you did not write it, so it does not sense to disturb you :)